end-to-end-tests/specs/home.spec.js

```diff
@@ -0,0 +1,93 @@
+var assert = require('assert');
+
+const CBIOPORTAL_URL = process.env.CBIOPORTAL_URL.replace(/\/$/, "");
+
+describe('homepage', function() {
+
+    it('it should show dev mode when testing', function() {
+        browser.url(CBIOPORTAL_URL);
+
+        browser.localStorage('POST', {key: 'localdev', value: 'true'});
+        browser.refresh();
+
+        var devMode = $('.alert-warning');
+
+        devMode.waitForExist(30000);
```

Comment:
@alisman It's 30,000 so 30s. Travis can be quite slow (especially the startup), this number is based on experimentation. I think it uses default timeout if you don't put it a number, which is configured in the webdriver conf.